narf/combineutils.py

```diff
@@ -79,10 +79,12 @@ def __init__(self, filename):
             self.noigroups = f['hnoigroups'][...]
             self.noigroupidxs = f['hnoigroupidxs'][...]
             self.maskedchans = f['hmaskedchans'][...]
+            self.pseudodatanames = f['hpseudodatanames'][...].astype(str)
```

Comment:
This probably needs to be protected otherwise it will crash when running on hdf5 input without pseudodata present (e.g. from text2hdf5)